app/__init__.py

```diff
@@ -1,7 +1,29 @@
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from dotenv import load_dotenv
+import os
 
+db = SQLAlchemy()
+migrate = Migrate()
+load_dotenv()
 
 def create_app(test_config=None):
     app = Flask(__name__)
+    app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+    if not test_config: 
+        app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get("SQLALCHEMY_DATABASE_URI")
 
-    return app
+    else:
+        print("Testing is on")
```

Comment:
Was this for testing your code? Or was this intentional so that you know if you are testing?